src/localplayer.h

```diff
@@ -143,6 +141,9 @@ class LocalPlayer : public Player
 
 	void setCollisionbox(const aabb3f &box) { m_collisionbox = box; }
 
+	bool getCanZoom() const { return m_can_zoom; }
+	void setCanZoom(bool can_zoom) { m_can_zoom = can_zoom; }
```

Comment:
setCanZoom -> permitZoom

Reply:
no, setCanZoom is better as it works either way

Reply:
'set' is consistent with all other 'setters', it would not be a 'setter' otherwise.`
'permit' is your incorrect use of English, you use it a lot where you shouldn't :)